packages/reactivity/src/effect.ts

```diff
@@ -83,7 +87,24 @@ export class ReactiveEffect<T = any> {
     recordEffectScope(this, scope)
   }
 
+  pause() {
+    this._isPaused = true
+  }
+
+  resume(runOnce = false) {
```

Comment:
```suggestion
  resume(immediate = false) {
```

I found that `runOnce` is a but confusing, maybe `immediate`? I think it's better to have JSDocs to explain things

Reply:
Yes your suggestion is perfect, I will modify it immediately.